### PR TITLE
R template

### PR DESCRIPTION
**Pull request by thomcsmits**

### CODE REVIEW

File: src/user_templates_api/templates/jupyter_lab/templates/blank_r/metadata.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (john-conroy) [inline src/user_templates_api/templates/jupyter_lab/templates/blank_r/metadata.json:1]: Should we add a `job_types` field which is an array of the compatible job type ids from https://workspaces.api.hubmapconsortium.org/job_types/? We can specify them for the non-jupyter_lab templates and backfill after the fact?

File: src/user_templates_api/templates/jupyter_lab/templates/blank/metadata.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-    "title": "Blank notebook",
-    "description": "Notebook only inserting UUIDs of added datasets",
+    "title": "Blank Notebook in Python",
```
Comment (john-conroy) [inline src/user_templates_api/templates/jupyter_lab/templates/blank/metadata.json:2]: If we add a distinguishing field like `job_types` we wouldn't need to change the title.

**Comment by thomcsmits** [comment]
> @jpuerto-psc what would the call be to `/templates/{template_type}/{template_name}/` for this template?

I believe it's still `/templates/jupyter_lab/blank_r/` (I'm able to retrieve it this way)

**Comment by jpuerto-psc** [comment]
@thomcsmits is correct

**Comment by john-conroy** [comment]
Do we expect that behavior to change at any point? For now we can assume the `template_type` will always be `jupyter_lab`?

**Comment by john-conroy** [comment]
Asking Tiffany to clarify whether we should distinguish the environments using the title as it is here or using the `job_types` to do so in the portal-ui.

File: src/user_templates_api/templates/jupyter_lab/templates/blank/metadata.json (hunk 2 above)
Comment (john-conroy) [inline src/user_templates_api/templates/jupyter_lab/templates/blank/metadata.json:2]:
```suggestion
    "title": "Blank Notebook",
```

File: src/user_templates_api/templates/jupyter_lab/templates/blank_r/metadata.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  "title": "Blank Notebook in R",
```
Comment (john-conroy) [inline src/user_templates_api/templates/jupyter_lab/templates/blank_r/metadata.json:2]:
```suggestion
  "title": "Blank Notebook",
```

**Review by john-conroy — CHANGES_REQUESTED** [review]
I talked to Tiffany. We're going to rely on `job_types` to differentiate templates.

**Review by john-conroy — APPROVED** [review]
Should be good to go now. Thanks!

**Pull request merged**